### PR TITLE
Release 1.0.1

### PR DESCRIPTION
**Pull request by adam-in-ict**
Updated package.json to match changes made to build.yml workflow and fix NPM package creation and publishing.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+  "version": "1.0.1",
   "description": "A JupiterOne Integration",
   "license": "MPL-2.0",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "src/index.js",
+  "types": "src/index.d.ts",
   "files": [
-    "dist"
+    "src"
```
Comment (VDubber) [inline package.json:9]: any questions in the /jupiterone dir?

Comment (VDubber) [reply package.json:9]: graph-github specifies jupiterone here..

**Review by VDubber — DISMISSED** [review]
Looks good! Added a comment that may or may not apply.

File: package.json (hunk 1 above)
Comment (adam-in-ict) [reply package.json:9]: Crap.  Good catch.  That'll bite me shortly if I don't add a questions file.  Thanks!

File: jupiterone/questions/questions.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+sourceId: managed:template
+integrationDefinitionId: '${integration_definition_id}'
+questions: []
```
Comment (zemberdotnet) [inline jupiterone/questions/questions.yaml:4]: I would pull in the `questions.yaml` from the integration project. They will be overwritten when bundling, so they should be published here or we'll lose them

File: package.json [hunk 3]
```diff
@@ -15,15 +16,17 @@
     "start": "j1-integration collect",
     "graph": "j1-integration visualize",
     "graph:types": "j1-integration visualize-types",
+    "validate:questions:dry": "j1-integration validate-question-file --dry-run",
+    "validate:questions": "j1-integration validate-question-file -a $MANAGED_QUESTIONS_JUPITERONE_ACCOUNT_ID -k $MANAGED_QUESTIONS_JUPITERONE_API_KEY",
     "lint": "eslint . --cache --fix --ext .ts,.tsx",
     "format": "prettier --write '**/*.{ts,js,json,css,md,yml}'",
+    "format:check": "prettier --check '**/*.{ts,js,json,css,md,yml}'",
```
Comment (VDubber) [inline package.json:23]: 💯

**Review by VDubber — APPROVED** [review]

**Pull request merged**